modules/logging-rn-5.7.1.adoc

```diff
@@ -0,0 +1,27 @@
+//module included in logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="logging-release-notes-5-7-1_{context}"]
+= Logging 5.7.1
+This release includes: link:https://access.redhat.com/errata/RHBA-2023:3197[OpenShift Logging Bug Fix Release 5.7.1].
+
+[id="logging-5-7-1-bug-fixes"]
```

Comment:
all module IDs need to include context per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-module-files